### DISCUSSION TITLE
Wrong GPU dedicated bytes counter?

### DISCUSSION THREAD

**Discussion by 0x4E69676874466F78** [winsiderss/systeminformer#1100] [Questions / Answers] [answered]
![изображение](https://user-images.githubusercontent.com/4449851/149928951-1492780e-eb07-44d4-bf93-e614fe72c7f2.png)
![изображение](https://user-images.githubusercontent.com/4449851/149930022-34f1de75-6834-4466-9a70-fdcb75bf8c44.png)

I restarted process hacker and it's still 3.1GB. Is windows/driver lying or am I misunderstanding something?

**Comment by dmex** [comment, marked as answer]
> Is windows/driver lying

Yes... Microsoft has busted the GPU counters again.

Task Manager is also showing these bogus values:

![image](https://user-images.githubusercontent.com/1306177/150074263-41a138d7-960a-4b1e-b069-508ea5bc53c4.png)

![image](https://user-images.githubusercontent.com/1306177/150074310-943ef816-f343-49df-978f-5d8ce3dd73f5.png)

**Reply by 0x4E69676874466F78** [reply]
Oh, this bug is even in windows 11
